packages/histoire-plugin-svelte/src/index.ts

```diff
@@ -28,7 +28,10 @@ export function HstSvelte (): Plugin {
           },
         },
         viteIgnorePlugins: [
-          'vite-plugin-svelte-kit',
+          // svelte-kit vite plugins
+          // reference: https://github.com/sveltejs/kit/blob/master/packages/kit/src/exports/vite/index.js
+          'vite-plugin-sveltekit-build',
+          'vite-plugin-sveltekit-middleware',
```

Comment:
The two plugins have been renamed to `vite-plugin-sveltekit-setup` and `vite-plugin-sveltekit-compile` in the latest version. Sorry for the breaking change. I wanted to sneak this in before SvelteKit 1.0.

We made this change working together with the Storybook team, so I'm hoping it will help Histoire as well. The idea is that you should be able to leave the setup plugin in place to setup SvelteKit's aliases, etc. and just remove the compile plugin. The Storybook folks said that's working for them in dev mode and almost working in build mode, but currently failing so we need to tweak a couple lines in SvelteKit to fix build mode, but hopefully the plugin names should have settled down at least now.

Reply:
Thanks for the work @benmccann. I'll adjust the PR